s3/src/test/scala/akka/stream/alpakka/s3/impl/S3NoMock.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.s3.impl
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.ContentTypes
+import akka.stream.ActorMaterializer
+import akka.stream.alpakka.s3.auth.AWSCredentials
+import akka.stream.alpakka.s3.scaladsl.S3Client
+import akka.stream.scaladsl.{Sink, Source}
+import akka.util.ByteString
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{BeforeAndAfterAll, FlatSpecLike, Matchers}
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+/**
+ *
+ * @author siva
+ */
```

Comment:
Please don't use author tags, we encourage collective code ownership: https://github.com/akka/alpakka/blob/master/CONTRIBUTING.md#pull-request-requirements